app/src/main/java/org/xwiki/android/sync/AppContext.kt

```diff
@@ -59,7 +62,15 @@ lateinit var userAccountsCookiesRepo: UserAccountsCookiesRepository
 private val apiManagers: MutableMap<UserAccountId, BaseApiManager> = mutableMapOf()
 
 fun resolveApiManager(serverAddress: String, userAccountId: UserAccountId): BaseApiManager = apiManagers.getOrPut(userAccountId) {
-    BaseApiManager(serverAddress, userAccountId, userAccountsCookiesRepo)
+    runBlocking {
```

Comment:
it would be better to make this method `suspend` instead of using `runBlocking`. Otherwise. it must be replaced with some other blocking function